### PR TITLE
feat(docs): Guide - Background Removal

### LINKED ISSUE #18: [Feature] Guide - Background Removal
# **Feature Request**

## **Is your feature request related to a problem? Please describe.**

Create a new guide in the documentation that details Background Removal

## **Describe the solution you'd like**

This guide should start off as a copy of the Next.js version: https://next.cloudinary.dev/guides/background-removal

The markdown/MDX can be found in the Next.js repo to make it easier to copy over: https://github.com/cloudinary-community/next-cloudinary/blob/main/docs/pages/guides/background-removal.mdx
Once the primary document is copied over:
- Update the components based on the Astro Cloudinary documentation usage (see other documentation pages)
- Update any usage patterns to reference Astro Cloudinary components
- Suggest any improvements to the guide to make it easier for people to understand and provide SEO value

This guide is lacking in depth, please be sure to include more content that can help people understand how to use the tools provided by the SDK.

### PR DESCRIPTION
**Pull request by smv1256**
# Description

Created a background removal guide with examples.

## Issue Ticket Number

Fixes #18 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Fix or improve the documentation
- [ ] This change requires a documentation update

# Checklist

- [x] I have followed the contributing guidelines of this project as mentioned in [CONTRIBUTING.md](/CONTRIBUTING.md)
- [x] I have created an [issue](https://github.com/cloudinary-community/astro-cloudinary/issues) ticket for this PR
- [x] I have checked to ensure there aren't other open [Pull Requests](https://github.com/cloudinary-community/astro-cloudinary/pulls) for the same update/change?
- [x] I have performed a self-review of my own code
- [ ] I have run tests locally to ensure they all pass
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes needed to the documentation

### CODE REVIEW

**Comment by kcoderhtml** [comment]
Glad you were able to write it :) i kept meaning to finish it but never got around to it

File: docs/src/content/docs/guides/background-removal.mdx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    width="960"
+    height="600"
+    preserveTransformations
+    alt=""
+    sizes="50vw"
+  />
+</HeaderImage>
+
+:::note
+  You must use `removeBackground` before `background` or `underlay` to remove the initial background.
```
Comment (colbyfayock) [inline docs/src/content/docs/guides/background-removal.mdx:114]: you actually dont need to worry about this here, it'll be addressed downstream in the code, so someone could use the props how they'd like

**Comment by colbyfayock** [comment]
hey @smv1256 thanks for putting this together, did you happen to check this out locally?

looks like there are some formatting issues with the Tip here

image

https://astro-cloudinary-git-fork-smv1256-main-cloudinarydevx.vercel.app/guides/background-removal

**Comment by smv1256** [comment]
@colbyfayock thank you so much for reviewing! Updated. Apologies for the errors--I'm new to open-source 😅

**Comment by colbyfayock** [comment]
hey @smv1256 a few more tweaks

"Background Removal in Astro"

Can you update that to "Remove a Background from an Image in Astro" or "Remove Backgrounds from Images in Astro"

the link here doesn't work
image

can you add it to the sidebar? you do this in the astro config file

image

**Comment by colbyfayock** [comment]
@smv1256 thanks for your contribution. this PR is eligible for free Cloudinary Hackatoberfest swag. Please send me an email at [hacktoberfest@cloudinary.com](mailto:hacktoberfest@cloudinary.com) with your name, GitHub username, and a link to the PR where I'll send further instructions.

Happy Hacktoberfest!

**Comment by colbyfayock** [comment]
@all-contributors please add @smv1256 for doc

**Pull request merged**